packages/cli/src/commands/pg/settings/auto-explain/log-min-duration.ts

```diff
@@ -11,8 +11,8 @@ export default class LogMinDuration extends PGSettingsCommand {
   `)
 
   static args = {
-    database: Args.string(),
-    value: Args.string(),
+    database: Args.string({description: 'The config var exposed to the owning app containing the database configuration.'}),
+    value: Args.string({description: 'Number indicating the min duration in milliseconds for queries before execution plans will be logged. A value of -1 disables it. A value of 0 means all queries will have their execution plans logged.'}),
```

Comment:
Change to "minimum duration in milliseconds for queries before logging execution plans. A value of -1 disables it. A value of 0 logs all query execution plans."